packages/back-end/package.json

```diff
@@ -11,7 +11,7 @@
     "build:typescript": "tsc",
     "dev": "yarn build:clean && concurrently \"yarn watch-compile\" \"yarn watch-dev | yarn pino-pretty\"",
     "build": "yarn build:clean && yarn build:typescript && yarn build:emails",
-    "start": "node dist/server.js",
+    "start": "TS_NODE_BASEURL=./dist node -r tsconfig-paths/register dist/server.js",
```

Comment:
Curious about this change. Is there a way to test your changes to see this fail, and then adding this to see it succeed? e.g. is it reproducible when building the docker container for production? Or is this something we can only tell when we ship to production?

Reply:
Yes - the way I tested it was to build the production image locally (by running `docker build .` in project root) and subsequently running the image to see if the app would boot with no issues. It would fail without including `tsconfig-paths`, which was expected ... Once added, along w/ the `TS_NODE_BASEFUL` env var, the backend would boot successfully.